lib/node_modules/@stdlib/math/base/special/acosd/docs/types/index.d.ts

```diff
@@ -33,11 +33,15 @@
 * // returns ~60.0
 *
 * @example
-* var v = acosd( Math.sqrt( 2 ) / 2 );
+* var sqrtf = require( '@stdlib/math/base/special/sqrtf' );
+*
+* var v = acosd( sqrtf( 2 ) / 2 );
 * // returns ~45.0
 *
 * @example
-* var v = acosd( Math.sqrt( 3 ) / 2 );
+* var sqrtf = require( '@stdlib/math/base/special/sqrtf' );
+*
+* var v = acosd( sqrtf( 3 ) / 2 );
 * // returns ~30.0
 *
 * @example
```

Comment:
@aayush0325  Looks like you've made the changes in `acosd` instead of `acosdf`.

Reply:
I am so incredibly sorry

Reply:
reverted that commit and added new examples using `sqrtf` wherever needed